src/test/java/com/rultor/agents/daemons/TailITCase.java

```diff
@@ -0,0 +1,106 @@
+/**
+ * Copyright (c) 2009-2015, rultor.com
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met: 1) Redistributions of source code must retain the above
+ * copyright notice, this list of conditions and the following
+ * disclaimer. 2) Redistributions in binary form must reproduce the above
+ * copyright notice, this list of conditions and the following
+ * disclaimer in the documentation and/or other materials provided
+ * with the distribution. 3) Neither the name of the rultor.com nor
+ * the names of its contributors may be used to endorse or promote
+ * products derived from this software without specific prior written
+ * permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT
+ * NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
+ * FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL
+ * THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+ * INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+ * HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+ * STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
+ * OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.rultor.agents.daemons;
+
+import com.google.common.base.Charsets;
+import com.jcabi.ssh.SSHD;
+import com.rultor.Time;
+import com.rultor.spi.Talk;
+import java.io.File;
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.IOUtils;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.xembly.Directives;
+
+/**
+ * Tests for ${@link Tail}.
+ *
+ * @author Armin Braun (me@obrown.io)
+ * @version $Id$
+ * @since 1.62
+ */
+public final class TailITCase {
+
+    /**
+     * Attribute key for the id of a talk directive.
+     */
+    private static final String DIRECTIVE_ID_KEY = "id";
+
+    /**
+     * Temp directory.
+     * @checkstyle VisibilityModifierCheck (5 lines)
+     */
+    @Rule
+    public final transient TemporaryFolder temp = new TemporaryFolder();
+
+    /**
+     * Tail can convert non UTF-8 chars in StdOut to UTF-8.
+     * @throws Exception In case of error.
+     */
+    @Test
+    public void tailsNonUtf() throws Exception {
+        final SSHD sshd = new SSHD(this.temp.newFolder());
+        final File home = new File(sshd.home(), "testTail");
+        FileUtils.forceMkdir(home);
+        final String clean = "some output";
+        FileUtils.write(
```

Comment:
@longtimeago phew :), in isolation I would agree here 100%, but given the code base we currently have I'd rather leave it like this.
If you look into the other tests in the same folder they all use FileUtils and follow a pretty common approach to setting things up.
Also I guess you'd agree that these tests are anything but straight forward to read already, given that we're in essence writing a JUnit test for a bash script here :D. 
So I'd rather keep them as homogeneous as possible (Just look at how few PRs Rultor already gets, anything to make things a little easier right ? :) ).
Also using Files.write adds another import and more verbosity to this thing, and it really needs anything but being more verbose.
=> I'd rather leave it at this unless have strong objections? :)